examples/todo-mvc/ui/widgets/selection_list_view.slint

```diff
@@ -91,6 +92,8 @@ export component SelectionListView inherits ListView {
     callback toggle(/* index */ int);
     callback remove(/* index */ int);
 
+    accessible-role: list;
```

Comment:
Just a reminder that the accessible role is not the right one. At least for the winit/AccessKit backend you are using the role for semantic lists, as you would use in a paragraph of text. The role to use is `Role::ListBox` and each element in the list should have `Role::ListBoxOption`.

Reply:
Ok, we don't have support for `ListBox` yet :(. Maybe not for this PR.

From look at the ARIA description for list vs. listbox, it sounds like to me that `ListView` should default to `listbox` as a role, as more often than not it's something the user interacts with and selects from, as opposed to a static `list` just for display (like a bullet list). Would you agree?

Reply:
Yes, that's right!